.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+www
```

Comment:
This is the directory that the generated Eleventy site will go into. Defined in the .eleventy.js#L7 file above.
This just prevents the generated site from getting committed into the repo.